pkg/config/config.go

```diff
@@ -314,6 +314,8 @@ type Update struct {
 	ReleaseMonitor *ReleaseMonitor `yaml:"release-monitor,omitempty"`
 	// The configuration block for updates tracked via the Github API
 	GitHubMonitor *GitHubMonitor `yaml:"github,omitempty"`
+	// The configuration block for transforming the `package.version` into an APK version
+	VersionTransform *VersionTransform `yaml:"version-transform,omitempty"`
```

Comment:
I would make it an array, that way multiple version transforms can stack.

```suggestion
	VersionTransform []VersionTransform `yaml:"version-transform,omitempty"`
```